app/services/notification_service.rb

```diff
@@ -2,7 +2,7 @@ class NotificationService
   class << self
     def send_instructeur_email_notification
       Instructeur
-        .includes(assign_to: { procedure: :dossiers })
+        .includes(assign_to: [ :procedure ])
```

Comment:
```suggestion
        .includes(assign_to: :procedure)
```

ça marche aussi, non ?

Reply:
Oué, ça marche aussi, mais je trouve ça pas si clair : c'est pas très explicite qu'on cherche à préloader `assign_to -> procedure`, et aussi dès qu'on veut rajouter une relation ou une sous-relation supplémentaire on doit rajouter les [] de tt façon, en googlant pour vérifier que c'est équivalent…

Cela dit si tu préfères la syntaxe plus brève, ça me va aussi.